packages/access-client/src/cli/index.js

```diff
@@ -31,111 +31,104 @@ prog
-    // } else {
-    //   console.error(`Run "${NAME} setup" first`)
-    // }
+    const store = new StoreConf({ profile: opts.profile })
+    const { url } = await getService(opts.env)
+    if (await store.exists()) {
+      const agent = await Agent.create({
+        store,
+        url,
+      })
+      const space = await selectSpace(agent)
+      try {
+        const result = await agent.getSpaceInfo(space)
+        console.log(result)
+      } catch (error_) {
+        const error = /** @type {Error} */ (error_)
+        console.log(error.message)
+      }
+    } else {
+      console.error(`Run "${NAME} setup" first`)
+    }
   })
 
 prog
   .command('delegate')
   .describe('Delegation capabilities.')
+  .option('--file', 'File to write the delegation into.')
   .action(async (opts) => {
-    // const store = new StoreConf({ profile: opts.profile })
-    // const { url } = await getService(opts.env)
-    // if (await store.exists()) {
-    //   const agent = await Agent.create({
-    //     store,
-    //     url,
-    //   })
-    //   const accountDids = agent.data.accounts.map((acc) => {
-    //     return { name: acc.did() }
-    //   })
-    //   const { account } = await inquirer.prompt([
-    //     {
-    //       type: 'list',
-    //       name: 'account',
-    //       choices: accountDids,
-    //       message: 'Select account:',
-    //     },
-    //   ])
-    //   const abilities = []
-    //   for (const [key, values] of agent.data.delegations.receivedByResource) {
-    //     if (key === account) {
-    //       for (const cap of values) {
-    //         abilities.push({ name: cap.cap.can })
-    //       }
-    //     }
-    //   }
-    //   const { ability } = await inquirer.prompt([
-    //     {
-    //       type: 'list',
-    //       name: 'ability',
-    //       choices: abilities,
-    //       message: 'Select ability:',
-    //     },
-    //   ])
-    //   const { audience } = await inquirer.prompt([
-    //     {
-    //       type: 'input',
-    //       name: 'audience',
-    //       choices: abilities,
-    //       message: 'Input audience:',
-    //     },
-    //   ])
-    //   console.log(account, ability)
-    //   const delegation = await agent.delegate(
-    //     Verifier.parse(audience),
-    //     [
-    //       {
-    //         can: ability,
-    //         with: account,
-    //       },
-    //     ],
-    //     800_000
-    //   )
-    //   console.log(await delegationToString(delegation))
-    // } else {
-    //   console.error(`Run "${NAME} setup" first`)
-    // }
+    const store = new StoreConf({ profile: opts.profile })
+    const { url } = await getService(opts.env)
+    if (await store.exists()) {
+      const agent = await Agent.create({
+        store,
+        url,
+      })
+      const space = await selectSpace(agent)
+
+      await agent.setCurrentSpace(space)
+
+      const { audience, expiration, name, type, abilities } =
+        await inquirer.prompt([
```

Comment:
Note: an interactive CLI means that folks can't use this for scripting...which I think will be an important use case initially since we won't have implementations in other languages.

Reply:
https://github.com/web3-storage/w3protocol/issues/241